policy/release/buildah_build_task.rego

```diff
@@ -20,6 +20,7 @@ import data.lib
 #   short_name: dockerfile_param_not_included
 #   failure_msg: DOCKERFILE param is not included in the task
 deny contains result if {
+	count(lib.pipelinerun_attestations) > 0
 	not dockerfile_param
```

Comment:
```suggestion
	buildah_task # there is a buildah task
	not dockerfile_param # ...with no dockerfile param
```

Reply:
Does it make sense? IIUC that will pass and fail in exactly the same way, but the intent is more clear.

Reply:
It does. Updated. Thanks!